lib/brcobranca/boleto/template/rghost_bolepix.rb

```diff
@@ -62,29 +70,32 @@ def method_missing(m, *args)
         def modelo_generico(boleto, options = {})
           doc = Document.new paper: :A4 # 210x297
 
-          template_path = File.join(File.dirname(__FILE__), '..', '..', 'arquivos', 'templates', 'modelo_generico.eps')
+          template_path = File.join(File.dirname(__FILE__), '..', '..', 'arquivos', 'templates', 'modelo_generico3.eps')
```

Comment:
```suggestion
          template_path = File.join(File.dirname(__FILE__), '..', '..', 'arquivos', 'templates', 'modelo_praso.eps')
```
Just as a suggestion, change the file name to not be generic model 3, it could be something like praso model.

Reply:
thanks 😸